### PR TITLE
 Time enabled and activated layers must not be added twice on topic change

### LINKED ISSUE #2720: Topic Switch and Default Layers
Step to reproduce:
1. Open map.geo.admin.ch
2. Switch to bafu to topic
3. Switch back to ech

Result:
Journey through time layer is added 2 times to the map and can been seen twice in the layer tree

Expected Result:
Journey through time is added only once.

### PR DESCRIPTION
**Pull request by oterral**
Fix #2720 

[Test](https://mf-geoadmin3.dev.bgdi.ch/teo_fix_2720/)

### CODE REVIEW

File: src/components/map/MapService.js [hunk 1]
```diff
@@ -1920,7 +1920,11 @@ goog.require('ga_urlutils_service');
               // Do not consider BOD layers that are already in the map,
               // except for timeEnabled layers
               var isOverlay = gaMapUtils.getMapOverlayForBodId(map, layerSpec);
-              if (bodLayer.timeEnabled || !isOverlay) {
+              // We test if timestamps existt to differentiate between topic
```
Comment (loicgasser) [inline src/components/map/MapService.js]: typo exist

File: src/components/map/MapService.js [hunk 2]
```diff
@@ -1920,7 +1920,11 @@ goog.require('ga_urlutils_service');
...
+              // selected layers and topic activated layers(no timestamps
```
Comment (loicgasser) [inline src/components/map/MapService.js]: layers(no timestamps -> space -> layers (no timestamps

**Comment by loicgasser** [comment]
LGTM and works fine thanks!

**Comment by oterral** [comment]
Typo fixed

**Pull request merged**